app/assets/stylesheets/mobile/mobile.scss

```diff
@@ -26,11 +26,7 @@ code {
   }
 
 body {
-  background: {
-    image: image-url("mobile/hatched-light.jpg");
-    /* scale background image down for iOS retina display */
-    size: 200px;
-  }
+  background-color: $sidebars-sub-background;
```

Comment:
I tested it. I found it a bit light. I think mobile view needs more contrast than desktop, don't you?